src/futils.h

```diff
@@ -330,7 +330,7 @@ extern int git_futils_fake_symlink(const char *new, const char *old);
  */
 typedef struct {
 	struct timespec mtime;
-	git_off_t  size;
+	git_object_size_t size;
```

Comment:
Some file APIs now use `uint64_t`, some use `git_object_size_t`. Shouldn't this one here use `uint64_t`, as well?

Reply:
Yes, this probably makes sense.